### ISSUE TITLE
Remove the need for FFMPEG Setup 

### ISSUE THREAD

**Issue by 9gix** [insert-coin/CloudyPanelPlugin#25]
Since we are modifying the UE source code, we might as well move the FFMPEG.exe into the UE directly.Thus for game developer who uses the plugin doesn't have to copy the FFMPEG again when they want to use cloudygame.

**Comment by joelngwt** [comment]
You want to put the executable in the repository? It is 35mb.

**Comment by 9gix** [comment]
I'm not sure how UE managed exe dependencies. I guess they use build script is it?
To put it into https://github.com/insert-coin/UnrealEngine/tree/cloudygame/Engine/Binaries

**Comment by joelngwt** [comment]
They use `setup.bat` https://github.com/insert-coin/UnrealEngine/blob/cloudygame/Setup.bat
and
`GenerateProjectFiles.bat` https://github.com/insert-coin/UnrealEngine/blob/cloudygame/GenerateProjectFiles.bat

**Comment by 9gix** [comment]
Yes, this sounds like a good idea to have in the future continuation.
